tests/rule_based_profiler/bobby_user_workflow_fixture.py

```diff
@@ -22,26 +22,6 @@ def bobby_columnar_table_multi_batch():
         verbose_profiler_config = f.read()
 
     my_row_count_rule_expectation_configurations: List[ExpectationConfiguration] = [
-        ExpectationConfiguration(
-            **{
-                "expectation_type": "expect_table_row_count_to_be_between",
-                "kwargs": {
-                    "min_value": 6727,
-                    "max_value": 9940,
-                    "mostly": 1.0,
-                },
-                "meta": {
-                    "profiler_details": {
-                        "metric_configuration": {
-                            "metric_name": "table.row_count",
-                            "metric_domain_kwargs": {
-                                "batch_id": "021563e94d7866f395288f6e306aed9b"
-                            },
-                        }
-                    }
-                },
-            }
-        ),
```

Comment:
Why was this removed?

Reply:
@talagluck It was duplicated code.  The cleaned up version here clearly separates the one table expectation from 30 column expectations.